distributed/asyncio.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
```

Comment:
I would be ok with avoiding the import in `__init__.py`, but I'm not sure how @mrocklin feels about that.